main.go

```diff
@@ -9,17 +9,33 @@ import (
 )
 
 func main() {
-	data, err := ioutil.ReadAll(os.Stdin)
-	if err != nil {
-		fmt.Println(err)
-		return
+	var(
+		match       bool
+		contentToUse []byte
+	)
+
+	if len(os.Args) < 2 {
+		data, err := ioutil.ReadAll(os.Stdin)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		contentToUse = data
+	} else {
+		fileContent, readFileErr := ioutil.ReadFile(os.Args[1])
+		if readFileErr != nil {
+			fmt.Println(readFileErr)
+			return
+		}
+		contentToUse = fileContent
+		match = true
 	}
 
-	if string(data) == "" {
+	if string(contentToUse) == "" {
```

Comment:
Still renamed.

Reply:
I cant change that back because contentToUse is wether data or file. Changing it would mean that in-place sorting wouldnt be possible anymore

Reply:
Then you'll have to make it possible again.